Signal/src/views/OWSProgressView.h

```diff
@@ -0,0 +1,14 @@
+//
+//  Copyright (c) 2017 Open Whisper Systems. All rights reserved.
+//
+
+#pragma once
+
+#import <UIKit/UIKit.h>
+
+@interface OWSProgressView : UIView
+
+@property (nonatomic) UIColor *color;
+@property (nonatomic) CGFloat progress;
```

Comment:
I rolled my own progress view.  Previously we were "using" `FFCircularProgressView`, which was reasonable but:

* It had a large, squarish shape which didn't fit within the thin "pillbox" shape of our audio player view.
* On completion, it changed appearance to a play button which was appropriate for video but not, for example, images.